pwnlib/tubes/ssh.py

```diff
@@ -1804,20 +1804,53 @@ def unlink(self, file):
 
         return self.sftp.unlink(file)
 
-    def libs(self, remote, directory = None):
+    def libs(self, remote, directory = None, flatten = False):
         """Downloads the libraries referred to by a file.
 
         This is done by running ldd on the remote server, parsing the output
         and downloading the relevant files.
 
         The directory argument specified where to download the files. This defaults
-        to './$HOSTNAME' where $HOSTNAME is the hostname of the remote server."""
+        to './$HOSTNAME' where $HOSTNAME is the hostname of the remote server.
+
+        Arguments:
+            remote(str): Remote file path
+            directory(str): Output directory
+            flatten(bool): Flatten the file tree if True (defaults to False). If
+                there are duplicate filenames, fallback to unflattened structure.
+
+        Examples:
+            >>> s =  ssh(host='example.pwnme')
+            >>> s.cwd = f'{pwnlib.data.elf.path}/ssh_libs'
```

Comment:
Please create an `__init__.py` file and use `pwnlib.data.elf.ssh_libs.get("duplicate")` etc. below. Check the the `__init__.py` in one of the other sub folders. The cwd has to be writeable and it apparently isn't in CI, when you change to the pwnlib data directory.